scripts/py_matter_idl/matter_idl/zapxml/handlers/parsing.py

```diff
@@ -14,14 +14,35 @@
 
 from matter_idl.matter_idl_types import AccessPrivilege, Attribute, AttributeQuality, DataType, Field, FieldQuality
 
+SIGNED_INTEGER_TYPES = {
+    'int8s': 1,
+    'int16s': 2,
+    'int24s': 4,
+    'int32s': 4,
+    'int40s': 8,
+    'int48s': 8,
+    'int56s': 8,
+    'int64s': 8,
+    'temperature':  2,
```

Comment:
This one seems off from the list.

Could we somehow start sharing/reuse bits in scripts/py_matter_idl/matter_idl/generators/types.py ?
__CHIP_SIZED_TYPES__ there has a much longer list, including temperatore but not singeling out that name (although indeed temperature is the only signed one)